doc/whats_new/v1.3.rst

```diff
@@ -151,6 +151,14 @@ Changelog
     :pr:`123456` by :user:`Joe Bloggs <joeongithub>`.
     where 123456 is the *pull request* number, not the issue number.
 
+:mod:`sklearn.feature_selection`
+................................
+
+- |Enhancement| All selectors in :mod:`sklearn.feature_selection` will preserve
+  a DataFrame's dtype when transformed. :pr:`25102` by `Thomas Fan`_.
+
+- |Fix| :class:`feature_selection.SequentialFeatureSelector`'s `cv` parameter
+  now supports generators. :pr:`25973` by :user:`Yao Xiao <Charlie-XIAO>`.
```

Comment:
This change seems unrelated with the current PR.

Reply:
Oh, sorry about that, not sure why it happend.